week2/homework/src/index.js

```diff
@@ -1,3 +1,95 @@
 'use strict';
 
-// TODO: Write the homework code in this file
+const fs = require('fs');
+let command = process.argv[2];
+let value = process.argv[3];
+let listFile = fs.readFileSync('./data.json', 'utf8');
+let listArr = JSON.parse(listFile);
```

Comment:
one thing you should try - it helps a lot with thinking about the application - is use better names.
`listArr` doesn't help understanding what that is, `todoList` is maybe a better name.
same with `listFile`, maybe `todoJson` will be easier to understand what this is when you look at this code again in 3 months :)